frontend/src/components/maps/leaflet/MapProperties.test.tsx

```diff
@@ -55,11 +55,11 @@ const mockDetails: IParcelDetail = {
     address: {
       line1: '1234 mock Street',
       line2: 'N/A',
-      city: 'Victoria',
-      province: 'BC',
+      cityId: 0,
+      provinceId: 'BC',
```

Comment:
This is unrelated to my changes but it was failing the tests since the API response changed but this test wasn't updated

Reply:
Thanks for fixing